src/components/admin/services/statistics/Chart.jsx

```diff
@@ -13,14 +13,14 @@ import {
 
 const Chart = ({ title, data }) => {
   const chartData = Object.entries(data).map(([type, value]) => ({
-    type: ROLES[type].label,
+    type: ROLES[type]?.label || SIZES[type]?.label,
     value: value,
-    className: ROLES[type].className,
-    fill: ROLES[type].fill,
+    className: ROLES[type]?.className || SIZES[type]?.className,
+    fill: ROLES[type]?.fill || SIZES[type]?.fill,
```

Comment:
make a better way so that we dont have to check if the value exists before calling fill/label on it